finalproject.txt

```diff
@@ -0,0 +1,19 @@
+Project: Video Shop Database - Olivier Gabison
```

Comment:
I'll be using loops when the user wants to view all of the videos checked out, videos inputted, videos overdue, and videos not checked out.